DependencyInjection/Configuration.php

```diff
@@ -81,7 +81,7 @@ public function getConfigTreeBuilder()
                             ->booleanNode('strict')->defaultTrue()->end()
                             ->booleanNode('escaping')->defaultTrue()->end()
                             ->booleanNode('line_breaks')->defaultTrue()->end()
-                            ->integerNode('max_newlines')->defaultValue(3)->end()
+                            ->scalarNode('max_newlines')->defaultValue(3)->end()
```

Comment:
> scalarNode to integerNode

Nice commit message :)

`integerNode` is a Symfony v2.2 feature but this bundle must support the v2.1.